metadata-models/src/main/pegasus/com/linkedin/ermodelrelation/ERModelRelationshipProperties.pdl

```diff
@@ -0,0 +1,90 @@
+namespace com.linkedin.ermodelrelation
+
+import com.linkedin.common.AuditStamp
+import com.linkedin.common.Urn
+import com.linkedin.common.CustomProperties
+
+/**
+ * Properties associated with a ERModelRelationship
+ */
+@Aspect = {
+  "name": "erModelRelationshipProperties"
+}
+record ERModelRelationshipProperties includes CustomProperties {
+
+  /**
+   * Name of the ERModelRelation
+   */
+  @Searchable = {
+    "fieldType": "TEXT_PARTIAL",
+    "enableAutocomplete": true,
+    "boostScore": 10.0
+  }
+  name: string
+
+  /**
+  * First dataset in the erModelRelationship (no directionality)
+  */
+   @Relationship = {
+     "name": "ermodelrelationA",
+     "entityTypes": [ "dataset" ]
+   }
+  @Searchable = {
+    "fieldType": "TEXT_PARTIAL",
+    "enableAutocomplete": true,
+    "boostScore": 10.0
+  }
+  source: Urn
+
+  /**
+  * Second dataset in the erModelRelationship (no directionality)
+  */
+   @Relationship = {
+     "name": "ermodelrelationB",
+     "entityTypes": [ "dataset" ]
+   }
+  @Searchable = {
+    "fieldType": "TEXT_PARTIAL",
+    "enableAutocomplete": true,
+    "boostScore": 10.0
+  }
+  destination: Urn
+
+
+  /**
+   * ERModelRelationFieldMapping (in future we can make it an array)
+   */
+  relationshipfieldMappings: array [RelationshipFieldMapping]
```

Comment:
one more tiny one! would you mind updating the casing of this model to be `relationshipFieldMappings` with a capital "F"? then i checked the code and there's just a couple of places that would have to be changed to reflect then then it's good!